go/app/main.go

```diff
@@ -10,16 +10,34 @@ import (
 	"github.com/labstack/echo/v4"
 	"github.com/labstack/echo/v4/middleware"
 	"github.com/labstack/gommon/log"
+
+	"encoding/json"
+	"io"
+	"crypto/sha256"
+	"mime/multipart"
+	"strconv"
+	// "errors"
 )
 
 const (
 	ImgDir = "images"
+	ItemsFilePath = "./items.json"
 )
 
 type Response struct {
 	Message string `json:"message"`
 }
 
+type Item struct {
+	Name string `json:"name"`
+	Category string `json:"category"`
+	ImageName string `json:"image-name"`
```

Comment:
```suggestion
	ImageName string `json:"image_name"`
```

[nit] image name should be snake_case to follow the naming style defined in the document:
```
{"items": [{"name": "jacket", "category": "fashion", "image_name": "510824dfd4caed183a7a7cc2be80f24a5f5048e15b3b5338556d5bbd3f7bc267.jpg"}, ...]}
```

Reply:
こちらで修正しました！

[3ca474f](https://github.com/uta1018/mercari-build-training/pull/2/commits/3ca474ff3ddb0bf21eb934e659ba8155da359c0a)